packages/cli/src/commands/test.ts

```diff
@@ -233,10 +242,15 @@ export default class Test extends AuthCommand {
       reporters.forEach(r => r.onSchedulingDelayExceeded())
     })
 
-    runner.on(Events.CHECK_SUCCESSFUL, (checkRunId, check, result, links?: TestResultsShortLinks) => {
+    runner.on(Events.CHECK_SUCCESSFUL, async (checkRunId, check, result, links?: TestResultsShortLinks) => {
       if (result.hasFailures) {
         process.exitCode = 1
       }
+
+      if (updateSnapshots) {
+        await pullSnapshots(configDirectory, result.assets?.snapshots)
+      }
```

Comment:
Given we will write/update stuff in the local env, maybe we should ask a confirmation from the user at end of the test session

Reply:
I'm not so sure. I think that the whole point of `--update-snapshots` is to pull down the snapshots. I think that the user passing `--update-snapshots` is already enough of a confirmation to do this. I double checked and PWT also doesn't ask for additional confirmation.